fe/src/main/java/org/apache/doris/load/BrokerFileGroup.java

```diff
@@ -326,17 +352,33 @@ public void readFields(DataInput in) throws IOException {
                 fileFormat = Text.readString(in);
             }
         }
+        // columnsFromPath
+        if (Catalog.getCurrentCatalogJournalVersion() >= FeMetaVersion.VERSION_59) {
```

Comment:
I think there is no need to persist `columnsFromPath`. Because Load have already persist all Load statement, and will generate DataDescriptor when restart.